src/lib.rs

```diff
@@ -216,7 +289,7 @@ pub fn from_string(p: String) -> Result<String> {
 
 #[cfg(feature = "wasm")]
 #[wasm_bindgen]
-pub fn from_string(p: String) -> std::result::Result<String, JsValue> {
+pub fn from_string(p: String, options: &'_ Options<'_>) -> std::result::Result<String, JsValue> {
```

Comment:
Ah I make this mistake pretty frequently as well. `rust-analyzer` won't actually type check this function because it only runs with default features on. This should be broken as `Parser` is missing a field.

This isn't a big deal since WASM releases are done manually and are mostly a novelty. It's not necessary to fix it.